app/src/main/res/drawable/nft_background_card.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid
+        android:color = "#ffffff"
+        />
+</shape>
```

Comment:
I don't think you need this.. you can just use android:background="@color/white" directly.